drivers/eks/eks_driver.go

```diff
@@ -41,24 +41,43 @@ const (
 )
 
 var amiForRegionAndVersion = map[string]map[string]string{
+	"1.15": map[string]string{
+		"us-east-2":      "ami-0da2b71057dfd9557",
+		"us-east-1":      "ami-0dc7713312a7ec987",
+		"us-west-2":      "ami-0e710550577202c55",
+		"ap-east-1":      "ami-00649306f8926416e",
+		"ap-south-1":     "ami-04655e6da8951ba94",
+		"ap-northeast-2": "ami-03ffb5acc344c1163",
+		"ap-southeast-1": "ami-088e05a9da0f9443c",
+		"ap-southeast-2": "ami-03f56833860168a9c",
+		"ap-northeast-1": "ami-0a1fe8c2cd382c5ec",
```

Comment:
can we move it above `ap-northeast-2` so it's alphabetical with AP region and matches the site?